modules/machine-health-checks-resource.adoc

```diff
@@ -7,28 +7,60 @@
 
 The MachineHealthCheck resource resembles the following YAML file:
 
+// Olde sample
```

Comment:
why is this commented out? can it be removed

Reply:
Yes; I believe Max had left it in there for reference during the updating process. I'll remove since it isn't necessary to have.